base-notebook/hooks/run_hook

```diff
@@ -1,17 +1,11 @@
 #!/bin/bash
 set -e
 
-# Apply tags
 GIT_SHA_TAG=${GITHUB_SHA:0:12}
-docker tag $IMAGE_NAME "$DOCKER_REPO:$GIT_SHA_TAG"
 PY_VERSION_TAG="python-$(docker run --rm ${IMAGE_NAME} python --version 2>&1 | awk '{print $2}')"
-docker tag $IMAGE_NAME "$DOCKER_REPO:$PY_VERSION_TAG"
 NB_VERSION_TAG="notebook-$(docker run --rm -a STDOUT ${IMAGE_NAME} jupyter-notebook --version | tr -d '\r')"
-docker tag $IMAGE_NAME "$DOCKER_REPO:${NB_VERSION_TAG%% }"
 LAB_VERSION_TAG="lab-$(docker run --rm -a STDOUT ${IMAGE_NAME} jupyter-lab --version | tr -d '\r')"
-docker tag $IMAGE_NAME "$DOCKER_REPO:${LAB_VERSION_TAG%%\r}"
 HUB_VERSION_TAG="hub-$(docker run --rm -a STDOUT ${IMAGE_NAME} jupyterhub --version | tr -d '\r')"
```

Comment:
These variables are used below, that's why I can't easily get rid of them, so I'll have to deal with manifests in this PR.